src/layer.ts

```diff
@@ -48,6 +52,13 @@ export const runtimeLookup: { [key: string]: RuntimeType } = {
   "python3.7": RuntimeType.PYTHON,
```

Comment:
I think while we're in here we can safely remove support for Node 10, Python 2.7, and Ruby 2.5, since it is no longer even possible to update a function that uses one of these runtimes. See:

https://docs.aws.amazon.com/lambda/latest/dg/runtime-support-policy.html

I don't think this should be considered a major/breaking change since these runtimes already won't work.